File: 新tablefun-miniprogram/replace-avatars.js
const fs = require('fs');

const filePath = 'c:/Users/夏雪/Desktop/新tablefun-miniprogram/pages/script-detail/script-detail.js';
let content = fs.readFileSync(filePath, 'utf-8');

// 替换所有 avatar[1-9].png 为 default-avatar.png
content = content.replace(/\/images\/avatar\d+\.png/g, '/images/default-avatar.png');

fs.writeFileSync(filePath, content, 'utf-8');
console.log('替换完成');

File: 新tablefun-miniprogram/replace-avatars.py
import re
import os

file_path = 'c:/Users/夏雪/Desktop/新tablefun-miniprogram/pages/script-detail/script-detail.js'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

content = re.sub(r'/images/avatar\d+\.png', '/images/default-avatar.png', content)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print('替换完成')
